Applications/Spire/Source/KeyBindings/KeyBindingsWindow.cpp

```diff
@@ -1,7 +1,8 @@
-#include "Spire/KeyBindings/KeyBindingsWindow.hpp"
+#include <QLabel>
 #include <QKeyEvent>
 #include <QVBoxLayout>
 #include "Spire/KeyBindings/CancelKeyBindingsTableView.hpp"
+#include "Spire/KeyBindings/KeyBindingsWindow.hpp"
```

Comment:
The include for a source file's header always goes first.

Reply:
According to our coding guidelines, include files are ordered based on their category where the first category is standard C++ header files, the second category is external dependency header files, and finally the third category is project header files. It seems to conflict with what you said.

Reply:
@kranar Yeah, it looks the style guide doesn't mention this: https://wiki.spiretrading.com/index.php/C%2B%2B_Style_Guide

@shaobinspire If a source file has a corresponding header, then that header is included first.

Reply:
Good, I can update that.